packages/commons/src/big-query/liquidity-indices-table/pull-data/pullClosestIndices.ts

```diff
@@ -0,0 +1,54 @@
+import { getBigQuery } from '../../client';
+import { getTableFullName } from '../../utils/getTableName';
+import { TableType } from '../../types';
+import { Address } from '../../../utils/convertLowercase';
+
+export type LiquidityIndexEntry = {
+    chainId: number;
+    rateOracle: Address,
+    liquidityIndex: number,
+    blockTimestamp: number
+}
+
+const mapToLiquidityIndices = (row: any) : LiquidityIndexEntry => row;
+
+export const pullClosestIndices = async (
+  chainId: number,
+  rateOracle: Address,
+  targetTimestamp: number
+): Promise<LiquidityIndexEntry[] | null> => {
```

Comment:
I'd suggest to change type to `Promise<[LiquidityIndexEntry, LiquidityIndexEntry] | null>`.

Reply:
as you mentioned below, this function can return 1 or 2 rows (even 0 if doesn't find the chainId or oracle). I left it like such and run sone checks in `getLiquidityIndexAt()`.